api/http/api_inventory.go

```diff
@@ -50,6 +50,7 @@ const (
 	uriInternalTenants       = "/api/internal/v1/inventory/tenants"
 	uriInternalDevices       = "/api/internal/v1/inventory/devices"
 	urlInternalDevicesStatus = "/api/internal/v1/inventory/tenants/:tenant_id/devices/:status"
+	uriInternalDeviceGroup   = "/api/internal/v1/inventory/tenants/:tenant_id/devices/:status/group"
```

Comment:
Why `:status`? Cannot it be `:device_id`?

Reply:
it cannot be device_id because we are already using "/api/internal/v1/inventory/tenants/:tenant_id/devices/:status"
if we'll use device_id it will cause runtime error:
 _inventory API handlers setup failed: failed to create router: Routes sharing a common placeholder MUST name it consistently: status != device_id_

Reply:
I can change /api/internal/v1/inventory/tenants/:tenant_id/devices/:status to /api/internal/v1/inventory/tenants/:tenant_id/devices/status/:status or /api/internal/v1/inventory/tenants/:tenant_id/devices/status and move status to the body
update workflows or workflows and deviceauth
and then use device_id here
/api/internal/v1/inventory/tenants/:tenant_id/devices/:device_id/group

Reply:
Oh, I didn't know the framework wants to have common prefixes.
I don't think moving the internal endpoint is a problem, but it is additional work.
Feel free to create a ticket for it, we can follow-up later without blocking your work in this PR.